spec/controllers/games_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe GamesController, type: :controller do
+
+  describe "games#index action" do
+    it "should successfully show the page" do 
+      get :index
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "games#new action" do 
+    it "should sucessfully show the new form" do
+      get :new
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "games#show action" do 
+    it "should successfully show the page if the game is found" do
+      game = FactoryGirl.create(:game)
```

Comment:
FYI, I've configured RSpec in such a way that you can simply use `create(:game)` here without the module name. But this is totally fine too. :)